.github/workflows/build_test_deploy.yml

```diff
@@ -506,7 +506,7 @@ jobs:
           name: next-swc-test-binary
           path: packages/next-swc/native
 
-      - run: echo "::set-output name=CNA_CHANGE::$(node scripts/run-for-change.js --type cna --always-canary --exec echo 'yup')"
+      - run: echo "CNA_CHANGE<<EOF" >> $GITHUB_OUTPUT; echo "$(node scripts/run-for-change.js --type cna --always-canary --exec echo 'yup')" >> $GITHUB_OUTPUT; echo "EOF" >> $GITHUB_OUTPUT
```

Comment:
Did you intend to have two `EOF` here?

Reply:
Two as in the beginning one and end one?

Reply:
Yeah. I guess I dont understand how the new format works for newlines like this

Reply:
Oh followed this tip to resolve the new line handling https://stackoverflow.com/a/74256214